File: Project/Frontend/daariiou-photography-frontend/src/api/lib/models/kind-of-shooting.ts
/* tslint:disable */
/* eslint-disable */
import { Picture } from './picture';
import { Shooting } from './shooting';
export interface KindOfShooting {
  koSid?: number;
  name?: null | string;
  pictures?: null | Array<Picture>;
  shootings?: null | Array<Shooting>;
}

File: Project/Backend/daariiou_photography_backend/daariiou_photography_backend/Models/KindOfShooting.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace daariiou_photography_backend.Models
{
    public partial class KindOfShooting
    {
        public KindOfShooting()
        {
            Pictures = new HashSet<Picture>();
            Shootings = new HashSet<Shooting>();
        }

        public int KoSid { get; set; }
        public string Name { get; set; }

        public virtual ICollection<Picture> Pictures { get; set; }
        public virtual ICollection<Shooting> Shootings { get; set; }
    }
}
